benchmark/syclblas/blas1/iamax.cpp

```diff
@@ -34,15 +34,15 @@ std::string get_name(int size) {
 }
 
 template <typename scalar_t>
-void run(benchmark::State& state, ExecutorType* executorPtr, index_t size,
+void run(benchmark::State& state, blas::SB_Handle* sb_handlePtr, index_t size,
```

Comment:
`sb_handlePtr` is mixing camel case and snake case. The file seems to mostly use camel case so maybe use `sb_handle_ptr`?